config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+class Config(object):
+    DEBUG = False
+    TESTING = False
+    DATABASE = ""
```

Comment:
add the "os.path.join(...)" here instead of in every controller; i noticed you have the same redundant code at the top of every controller you create, i dont see the reason for it, maybe you can explain me, otherwise it should be removed.